Describing scheduled and on-demand backups (#58)
Percona-Lab/percona-xtradb-cluster-operator: configure/backups.md

2. Now find out correct backup and cluster names. Available backups can be listed with the following command:
   ```
   kubectl get pxc-backup
   ```
   And the following command will list available clusters:
   ```
   kubectl get pxc
   ```
4. When correct names are known, the actual restoration process can be started as follows:
   ```
   ./deploy/backup/restore-backup.sh <backup-name> <cluster-name>
   ```

## Delete the unneeded backup

Deleting some previously saved backup needs not more than the backup name. This name can be taken from the list of available backups, returned by the following command:
   ```
   kubectl get pxc-backup
   ```

When the name is known, backup can be deleted as follows:

   ```
   kubectl delete pxc-backup/<backup-name>
   ```

## Copy backup to a local machine

Make a local copy of some previously saved backup needs not more than the backup name. This name can be taken from the list of available backups, returned by the following command:
   ```
   kubectl get pxc-backup
   ```

When the name is known, backup can be downloaded to the local machine as follows:
   ```
   ./deploy/backup/copy-backup.sh <backup-name> path/to/dir
   ```

> If needed, this downloaded backup can be restored to the local installation of Percona XtraDB Cluster:

Reviewer: ```suggestion
For example, this downloaded backup can be restored to the local installation of Percona Server:
```
